internal/tools/pipeline/providers/actionmgr/impl.go

```diff
@@ -61,7 +61,13 @@ func (s *provider) SearchActions(items []string, ops ...OpOption) (map[string]*d
 			}
 			diceYmlStr = string(rendered)
 		}
-		diceYml, err := diceyml.New([]byte(diceYmlStr), false)
+		var diceYml *diceyml.DiceYaml
+		var err error
+		if so.ClusterInfo != nil {
+			diceYml, err = diceyml.New([]byte(diceYmlStr), false, diceyml.WithPlatformInfo(so.ClusterInfo))
```

Comment:
how about pass nil map to diceyml parser?

Reply:
if pass nil map to diceyml and with platform visitor, `diceyml.New` will return error about image replaced-failure.


`SearchActions` was called in different funcs, only `taskop.Prepare` need replace image placeholder